libbeat/common/docker/watcher.go

```diff
@@ -447,3 +467,7 @@ func (w *watcher) ListenStart() bus.Listener {
 func (w *watcher) ListenStop() bus.Listener {
 	return w.bus.Subscribe("stop")
 }
+
+type systemClock struct{}
```

Comment:
Ah, I had in mind about moving this interface implementation too to the top 😬 .

Reply:
Ok, moved :slightly_smiling_face: